codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/ProtoToCpg.scala

```diff
@@ -0,0 +1,145 @@
+package io.shiftleft.codepropertygraph.cpgloading
+
+import java.lang.{Boolean => JBoolean, Integer => JInt, Long => JLong}
+import java.util.{NoSuchElementException, Collection => JCollection, Iterator => JIterator}
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.proto.cpg.Cpg.CpgStruct.{Edge, Node}
+import io.shiftleft.proto.cpg.Cpg.PropertyValue
+import org.apache.commons.configuration.Configuration
+import org.apache.logging.log4j.{LogManager, Logger}
+import org.apache.tinkerpop.gremlin.structure.{T, Vertex}
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerGraph
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ArrayBuffer
+
+object ProtoToCpg {
+  val logger: Logger = LogManager.getLogger(classOf[ProtoToCpg])
+
+  def addProperties(kvs: ArrayBuffer[AnyRef], name: String, value: PropertyValue): Unit = {
```

Comment:
Please try to avoid abbreviations like `kvs`. I suggest `keyValues`.
In general we try to use descriptive names even if they are a little bit longer.